src/antares/craft/model/output.py

```diff
@@ -9,9 +9,96 @@
 # SPDX-License-Identifier: MPL-2.0
 #
 # This file is part of the Antares project.
+from enum import Enum
+from typing import Any
+
+import pandas as pd
+
 from pydantic import BaseModel
 
 
+class QueryFile(Enum):
+    VALUES = "values"
+    DETAILS = "details"
+    DETAILS_STSTORAGE = "details-STstorage"
+    DETAILS_RES = "details-res"
+
+
+class Frequency(Enum):
+    HOURLY = "hourly"
+    DAILY = "daily"
+    WEEKLY = "weekly"
+    MONTHLY = "monthly"
+    ANNUAL = "annual"
+
+
+class McType(Enum):
+    ALL = "mc-all"
+    IND = "mc-ind"
+
+
+class ObjectType(Enum):
+    LINKS = "links"
+    AREAS = "areas"
+
+
+class AggregationEntry(BaseModel):
+    """
+    Represents an entry for aggregation queries
+
+    Attributes:
+        query_file: The file to query.
+        frequency: "hourly", "daily", "weekly", "monthly", "annual"
+        mc_years: Monte Carlo years to include in the query. If left empty, all years are included.
+        type_ids: which links/areas to be selected (ex: "be - fr"). If empty, all are selected (comma separated)
+        columns_names: names or regexes (if query_file is of type details) to select columns (comma separated)
+    """
+
+    query_file: QueryFile
+    frequency: Frequency
+    mc_years: str = ""
+    type_ids: str = ""
+    columns_names: str = ""
+
+    def to_query(self, object_type: ObjectType) -> str:
+        mc_years = f"&mc_years={self.mc_years}" if self.mc_years else ""
+        type_ids = f"&{object_type.value}_ids={self.type_ids}" if self.type_ids else ""
+        columns_names = f"&columns_names={self.columns_names}" if self.columns_names else ""
+
+        return f"query_file={self.query_file.value}&frequency={self.frequency.value}{mc_years}{type_ids}{columns_names}&format=csv"
+
+
 class Output(BaseModel):
     name: str
     archived: bool
+
+    def __init__(self, output_service, **kwargs: Any):  # type: ignore
```

Comment:
I think you should remove the usage of pydantic inside this class and just do a normal class